client/src/app/site/motions/services/motion-csv-export.service.ts

```diff
@@ -83,7 +87,12 @@ export class MotionCsvExportService {
      * @param contentToExport content properties to export
      * @param crMode
      */
-    public exportMotionList(motions: ViewMotion[], contentToExport: string[], crMode?: ChangeRecoMode): void {
+    public exportMotionList(
+        motions: ViewMotion[],
+        contentToExport: string[],
+        crMode?: ChangeRecoMode,
+        comments?: number[]
```

Comment:
both `crMode` and `comments` should be part of `contentToExport`.

If you don't want to fix this now, add a TODO